index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" type="image/png" href="https://www.stockport.gov.uk/assets/images/ui-images/sg/favicon-16x16.png" sizes="16x16">
-    <link rel="stylesheet" href="https://s3-eu-west-1.amazonaws.com/smbc-react-assets/int/styleguide/v2/styles.min.css" />
+    <link rel="stylesheet" href="https://design-system.stockport.gov.uk/int/1.0.0/smbc-frontend.min.css" />
```

Comment:
We should be targeting Major version 1 with no minor and patch in the URL, this will call out for version 1.0.0 of the DS but the corresponding changes in DS are version 1.2.2

Reply:
url has been updated to target major version only